ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.collection.js

```diff
@@ -66,6 +66,21 @@ module.exports = Backbone.AssociatedModel.extend({
         model: ResultForm,
         url: './internal/forms/result',
         initialize: function() {},
+        comparator: function(a, b) {
+          // Keep the new result form button at the front
```

Comment:
Yeah I did something similar for the new search form button. It also had a placeholder model in the collection so I removed it and hard coded a different view before rendering the collection.

I wanted to sort the result forms alphabetically without having to mess with the collection (since the original ticket was to add a dropdown list for the search forms) but I'll go ahead and make the same change here.